doc/CONTRIBUTING.md

```diff
@@ -14,3 +14,14 @@ Suggested branches naming convetion: `task_number-task_name`.
 Commit message naming convention: `task_number if implemented this commit will`.
 
 To get task numbers, use: [Trello Card Numbers Plus](https://chrome.google.com/webstore/detail/trello-card-numbers-plus/ncibjlmfhjcjnphnpphgphbflpdpliei/related)
+
+## Documentation
+
+The Swagger (OpenAPI 3) is used for API's documentation.
+The documentation can be found under `http://localhost:8080/swagger-ui/index.html` path by default.
+Note that API must be online if you are willing to check the documentation.
+
+Information about initializing Swagger can be found under [this link](https://springdoc.org).
+Information about usage and examples of it can be found under these links:
+1. [OpenAPI 3, documentation overview](https://www.youtube.com/watch?v=2o_3hjUPAfQ)
```

Comment:
The documentation alone should be informative enough to allow contributing without additional research, especially watching time-consuming YouTube videos. Therefore I suggest removing this link.